guides/source/configuring.md

```diff
@@ -54,149 +54,253 @@ NOTE: If you need to apply configuration directly to a class, use a [lazy load h
 
 These configuration methods are to be called on a `Rails::Railtie` object, such as a subclass of `Rails::Engine` or `Rails::Application`.
 
-* `config.after_initialize` takes a block which will be run _after_ Rails has finished initializing the application. That includes the initialization of the framework itself, engines, and all the application's initializers in `config/initializers`. Note that this block _will_ be run for rake tasks. Useful for configuring values set up by other initializers:
+#### config.after_initialize
```

Comment:
What do you think about retaining the backticks?  e.g.

```suggestion
#### `config.after_initialize`
```

Reply:
Sounds good!

Reply:
I've added the backticks.